README.md

```diff
@@ -5,22 +5,295 @@
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 This repository holds the source code for the Realm SDK for Flutter™ and Dart™.
 
-# Alpha
-
 **This project is in the Alpha stage, All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
 
-The previous preview version of Realm Dart can be found on this branch: https://github.com/realm/realm-dart/tree/preview
+## Getting Started
+
+* Import Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
+
+* Define a data model class `_Car`.
+    ```dart
+    @RealmModel()
+    class _Car {
+      late String make;
+
+      late String model;
+    
+      int? kilometers = 500;
+    }
+    ```
+
+* Generate Generates RealmObject class `Car` from data model class `_Car`.
+
+    ```
+    flutter run realm generate
+    ```
+
+* Open a Realm and add some objects.
+
+    ```dart
+    var config = Configuration([Car.schema]);
+    var realm = Realm(config);
+
+    var car = Car("Telsa", "Model Y", kilometers: 5);
+    realm.write(() {
+      realm.add(car);
+    }
+    ```
+
+* Query objects in Realm.
+
+    ```dart
+    var cars = realm.all<Car>();
+    Car myCar = cars[0];
+    print("My car is ${myCar.make} model ${myCar.model}");
+
+    cars = realm.all<Car>().query("make == 'Tesla'");
```

Comment:
Interesting: It seems the filter is working with both quotes combinations:
'make == "Tesla"' - this is how we are using it in tests
"make == 'Tesla'" - but this is more intuitive

Reply:
yes, this is parsed by core